packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -754,6 +754,24 @@ const ruleset: any = {
       },
     },
 
+    ///
+    /// ARM RPC rules for Arg
+    ///
+
+    // RPC Code: RPC-Arg-V1-01
+    ParametersSchemaAsTypeObject: {
+      rpcGuidelineCode: "RPC-POST-V1-05",
+      description: "The schema for body parameters must specify type:object and include a definition for its reference model.",
+      message: "{{description}}",
+      severity: "error",
+      resolved: true,
+      formats: [oas2],
+      given: "$.parameters.*.schema[?(@property === 'type' && @ !=='object')]",
```

Comment:
`parameters` should be under `paths.pathName.opName` (see my other comment)

Reply:
Addressed comment